source/pci/pci.go

```diff
@@ -106,7 +111,10 @@ func (s Source) Discover() (source.Features, error) {
 func readDevInfo(devPath string) (pciDeviceInfo, error) {
 	info := pciDeviceInfo{}
 
-	for _, attr := range devLabelAttrs {
+	for _, attr := range append(devLabelAttrs, extraDevAttrs...) {
+		if _, err := os.Stat(path.Join(devPath, attr)); os.IsNotExist(err) {
+			continue
+		}
```

Comment:
nit: I started to think if we should check whether the failing attribute belongs to extraDevAttrs or not. If one of the "mandatory" attributes is not found, we should prolly return an error. WDYT?

Reply:
In case of `sriov_totalvfs`, all PCI devices don't have it so I thought we could silently ignore if it's not there. `network` source is a good example of noisy logs:

`2020/02/17 14:50:16 SR-IOV not supported for network interface: eth0: open /sys/class/net/eth0/device/sriov_totalvfs: no such file or directory`

Reply:
> n case of `sriov_totalvfs`, all PCI devices don't have it so I thought we could silently ignore if it's not there

Yeah, sure, we can ignore that. But i was thinking about the others listed in `devLabelAttrs` that *should* not fail

Reply:
I misread your comment earlier. Yes, an error with mandatory attrs certainly makes sense.

Reply:
K, as I said they *should* not fail but prolly better be safe 😉

Reply:
I've updated the error handling. While debugging I realized that `ioutil.ReadFile` returns no error if the file is not available so the `if err != nil { ... }` has been *broken* all the time.

Reply:
> `ioutil.ReadFile` returns no error if the file is not available

Umm, waat?? Yes it does return an error, at least for me

Reply:
some glitch. nevermind